vigetlabs/wordpress-site-starter
# [#5] Logo Grid UI

# Summary

This PR pulls in Blueprint styles for the Logo Grid Component.

## Issues

- #5 

## Testing Instructions

1. Create a new page or edit an existing page
2. Insert the Logo Grid block
3. Upload photos to the image blocks inside of the component
4. Verify that:
   - Images resize correctly in the grid
   - Columns are responsive (two columns until `large` breakpoint, then four columns to match Blueprint)
   - Justification can be changed on the component and images will be positioned accordingly

## Screenshots

Two columns:

![Screenshot 2024-04-24 at 3 43 44 PM](https://github.com/vigetlabs/wordpress-site-starter/assets/2685062/cedea6fc-ee46-4dea-a245-f589d3afbe90)

Four columns: 

![Screenshot 2024-04-24 at 3 43 29 PM](https://github.com/vigetlabs/wordpress-site-starter/assets/2685062/a146afcc-1eb1-463f-9284-5cfa9e827517)

With Aspect Ratio:

![Screenshot 2024-04-25 at 10 50 55 AM](https://github.com/vigetlabs/wordpress-site-starter/assets/2685062/1ae9f9b6-3f88-48fe-9f9a-ab02701d0083)
